### PR TITLE
This Development-cycle in Cargo 1.79

### PR DESCRIPTION
**Pull request by epage**

### CODE REVIEW

File: posts/inside-rust/2024-05-07-this-development-cycle-in-cargo-1.79.md [hunk 1]
```diff
@@ -0,0 +1,458 @@
+---
+layout: post
+title: "This Development-cycle in Cargo: 1.79"
```
Comment (epage) [inline posts/inside-rust/2024-05-07-this-development-cycle-in-cargo-1.79.md:3]: I'm planning for this to be published on Tuesday, May 7th

File: posts/inside-rust/2024-05-07-this-development-cycle-in-cargo-1.79.md [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+
+Thanks to [LukeMathWalker](https://github.com/LukeMathWalker) for the suggestion!
+
+[Please submit your suggestions for the next post.](https://rust-lang.zulipchat.com/#narrow/stream/246057-t-cargo/topic/Plugin.20of.20the.20Dev.20Cycle/near/420703211)
+
+## Implementation
+
+##### Deprecations
+
+*[Update from 1.78](https://blog.rust-lang.org/inside-rust/2024/03/26/this-development-cycle-in-cargo-1.78.html#deprecated-cargotoml-fields)
```
Comment (ehuss) [inline posts/inside-rust/2024-05-07-this-development-cycle-in-cargo-1.79.md:52]:
```suggestion
*[Update from 1.78](https://blog.rust-lang.org/inside-rust/2024/03/26/this-development-cycle-in-cargo-1.78.html#deprecated-cargotoml-fields)*
```

File: posts/inside-rust/2024-05-07-this-development-cycle-in-cargo-1.79.md [hunk 3]
```diff
@@ -0,0 +1,458 @@
...
+
+## Focus areas without progress
+
+These are areas of interest for Cargo team members with no reportable progress for this development-cycle.
+
+Ready-to-develop:
+- [Merge `cargo upgrade` into `cargo update`](https://github.com/rust-lang/cargo/issues/12425)
+- [`cargo publish` for workspaces](https://github.com/rust-lang/cargo/issues/1169)
+- [Auto-generate completions](https://github.com/rust-lang/cargo/issues/6645)
+  - See [clap-rs/clap#3166](https://github.com/clap-rs/clap/issues/3166)
```
Comment (weihanglo) [inline posts/inside-rust/2024-05-07-this-development-cycle-in-cargo-1.79.md:400]: Isn't this already picked up by a GSoC participant? In this context it looks available to claim.

Comment (epage) [reply posts/inside-rust/2024-05-07-this-development-cycle-in-cargo-1.79.md]: Commented out this item

**Review by ehuss — APPROVED** [review]
Thanks!

**Pull request merged**